docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,171 @@
+# Release 465 (20 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in [EWKB
+  format](https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry)
+  with the {func}`ST_GeomFromBinary` function. ({issue}`23824`)
+* Add support for parameter of `bigint` type for the {func}`repeat` function. ({issue}`22867`)
+* Add support for the `ORDER BY` clause in a windowed aggregate function. ({issue}`23929`)
+* Expose query start count and rate metrics for resource groups. ({issue}`23984`)
+* {{breaking}} Update the required [](jvm-config). ({issue}`24161`)
+* {{breaking}} Change the data type for `client_info` in the MySQL event
+  listener to `MEDIUMTEXT`. ({issue}`22362`)
+* Improve performance of queries with selective joins. ({issue}`22824`)
+* Improve performance when using various string functions in queries involving
+  joins. ({issue}`24182`)
+* Fix potential configuration leakage between catalogs using JDBC-based
+  connectors. ({issue}`24058`)
+* Reduce chance of out of memory query failure when `retry-policy` is set to
+  `task`. ({issue}`24114`)
+* Prevent some query failures when `retry-policy` is set to `task`. ({issue}`24165`)
+
+## Server RPM
+
+* {{breaking}} Update the required [](jvm-config). ({issue}`24161`)
```

Comment:
See earlier comments.

Reply:
Here its actually a factual change on the config in the RPM (and container) so if people reuse and customize they should find out. I think we should at least have the entry .. maybe remove the breaking ..